.cirrus.yml

```diff
@@ -0,0 +1,66 @@
+env:
+  CIRRUS_CLONE_DEPTH: 1
+
+fedora_30_task:
+  container:
+    image: quay.io/avocado-framework/avocado-vt-ci-fedora-30
+  env:
+    PYTHON: python3
+    matrix:
+      - AVOCADO_SRC:
+      - AVOCADO_SRC: avocado-framework<70.0
+      - AVOCADO_SRC: -e git+https://github.com/avocado-framework/avocado#egg=avocado_framework
```

Comment:
It'd be nice to add single-line comments to those to explain them (default, LTS and latest-git) as I bet we'll forget what they stand for... But this could wait for later...